Sources/NIOHTTP1/HTTPDecoder.swift

```diff
@@ -256,7 +256,10 @@ private class BetterHTTPParser {
             self.finish { delegate, bytes in
                 delegate.didReceiveURL(bytes)
             }
-        case .beforeRequest, .headersComplete, .headerName, .trailerName, .trailerValue, .beforeURL:
+        case .afterMessageBegin:
```

Comment:
@Lukasa note, this is the real change: we're happy with `afterMessageBegin` (previously `.beforeURL`) iff we're parsing a response